backend/CLAUDE.md

```
@@ -0,0 +1,138 @@
+uv run pytest tests/unit/services/test_assistant_service.py
+
+# Run with verbose output
+uv run pytest -v
+
+# Run with coverage
+uv run pytest --cov=src --cov-report=html
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)